src/platform/cmx/chal/cpu_ghz.h

```diff
@@ -0,0 +1,3 @@
+#define CPU_GHZ 0.40
```

Comment:
For this to co-exist with x86, we might need to move this to the shared directory at config time.  Not very sure yet.